components/Location/Location.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image';
+import {
+  getFullDateString,
+  getRelativeTimeString,
+  getNameString
+} from '~/lib/utils';
+import styles from './style.module.css';
+import { useMemo } from 'react';
+
+const Location = ({ type, city, country, author, date }) => {
+  const name = useMemo(() => getNameString(author), [author]);
+  return (
+    <div className={styles.listings}>
+      <div className={styles.marker}>
+        <Image
+          src={`/images/markers/${type}-image.png`}
+          alt="Marker"
+          title={type}
+          width={70}
+          height={70}
+        />
+      </div>
+      <div className={styles.text}>
+        <b>
+          {' '}
+          {city}, {country}{' '}
+        </b>
+        <br></br>
+        {date && <i>{getRelativeTimeString(date)}</i>}
+        <br />
+        <span>{name}</span>
```

Comment:
I think it would be better here to wrap all this around `p` tags instead of those breaks.

Reply:
You mean like this?
![image](https://user-images.githubusercontent.com/80540164/188016752-cb0a49ca-6ada-4da1-9bd8-e12de011d397.png)
Cause it ends up looking very spaced out:
![image](https://user-images.githubusercontent.com/80540164/188016885-7a8e12a1-062c-4ee0-8e30-8dbd806c092e.png)

Reply:
Yes, and then you can just customize the padding with css.

You could also use <span> for each and then use something like css-flex on the div container.

I don't think it is a good practice to use br for layout purposes.

Reply:
Got it, solved it with css (`margin: 0`). Also made the city and country text a little bigger.